gravitee-apim-console-webui/src/management/settings/portal-ng/portal-ng.component.scss

```diff
@@ -0,0 +1,63 @@
+@use 'sass:map';
+@use '@angular/material' as mat;
+@use '@gravitee/ui-particles-angular' as gio;
+
+@use '../../../scss/gio-layout' as gio-layout;
+
+$typography: map.get(gio.$mat-theme, typography);
+
+:host {
+  @include gio-layout.gio-responsive-margin-container;
+}
+
+.portal {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+
+  &__form {
+    &__card {
+      margin-bottom: 16px;
+
+      &__title {
+        margin-top: 16px;
+        margin-bottom: 0;
+      }
+
+      &__radio-group {
+        display: flex;
+        flex-direction: column;
+        align-items: flex-start;
+      }
+
+      &__form-field {
+        width: 100%;
+      }
+
+      &__radio-tab {
+        display: flex;
+        flex-wrap: wrap;
+        justify-content: space-between;
+
+        &__group {
+          flex-basis: calc(50% - 8px);
+
+          &__radio-button {
+            min-height: 72px;
+            display: flex;
+            align-items: center;
+            justify-content: flex-start;
+            padding: 4px 0;
+          }
+        }
+      }
+    }
+  }
+}
+
+.title {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 16px 16px 0 16px;
```

Comment:
I'm not sure about that. I would expect all pages to have the same aligned title as on recently migrated pages like: api-logging

Reply:
Done